core/modules/query/ValueExpr.cc

```diff
@@ -82,6 +82,14 @@ std::ostream& operator<<(std::ostream& os, const ValueExpr::FactorOp& factorOp)
     case ValueExpr::MINUS: os << "MINUS"; break;
     case ValueExpr::MULTIPLY: os << "MULTIPLY"; break;
     case ValueExpr::DIVIDE: os << "DIVIDE"; break;
+    case ValueExpr::DIV: os << "DIV"; break;
+    case ValueExpr::MODULO: os << "MODULO"; break;
+    case ValueExpr::MOD: os << "MOD"; break;
+    case ValueExpr::BIT_SHIFT_LEFT: os << "BIT_SHIFT_LEFT"; break;
+    case ValueExpr::BIT_SHIFT_RIGHT: os << ">BIT_SHIFT_RIGHT>"; break;
+    case ValueExpr::BIT_AND: os << "BIT_AND"; break;
+    case ValueExpr::BIT_OR: os << "BIT_OR"; break;
+    case ValueExpr::BIT_XOR: os << "BIT_XOR"; break;
```

Comment:
To improve the readability of these repeating patters one could align them like tgis:
```
case ValueExpr::DIV:             os << "DIV";               break;
case ValueExpr::MODULO:          os << "MODULO";            break;
case ValueExpr::MOD:             os << "MOD";               break;
case ValueExpr::BIT_SHIFT_LEFT:  os << "BIT_SHIFT_LEFT";    break;
case ValueExpr::BIT_SHIFT_RIGHT: os << ">BIT_SHIFT_RIGHT>"; break;
case ValueExpr::BIT_AND:         os << "BIT_AND";           break;
case ValueExpr::BIT_OR:          os << "BIT_OR";            break;
case ValueExpr::BIT_XOR:         os << "BIT_XOR";           break;
```
ow you can see that `BIT_SHIFT_RIGHT` is represented differently than others with extra symbols at both ends `">BIT_SHIFT_RIGHT>"`. Is this by design?